src/app/scoreboard/scoreboard.component.html

```diff
@@ -1,15 +1,11 @@
 <section class="hero is-medium is-dark is-bold">
-  <div class="hero-body">
-    <!-- <div class="container"> -->
-      <div class="tile is-ancestor">
-        <div class="tile is-parent" *ngFor="let color of ['danger', 'info']; let i = index">
-          <app-team-tile
-            [color]="color"
-            (idSet)="idSet($event)"
-            (pointsChange)="pointsChanged($event)"
-            [currentServer]="server">
-          </app-team-tile>
```

Comment:
why was the formatting changed here?

Reply:
no idea I think it was a vscode thing...